packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -1763,45 +1747,38 @@ describeCompat("GC data store tombstone tests", "NoCompat", (getTestObjectProvid
 						resolveHandleHelper((summarizer2 as any).runtime, dataStore2._context.id),
 					`Should be able to request dataStore2 which is tombstoned in summarizer`,
 				);
-				// Summarize again. This should not result in any of the data stores getting un-tombstoned.
+
+				// Summarize again. This should result in the data stores getting un-tombstoned via auto-recovery.
 				const { summaryVersion: summaryVersion2 } = await summarize(summarizer2);
-				// There shouldn't be any revived event for dataStore3 because it is not revived.
-				// In the past, the DDS in dataStore2 was loaded and the handle to dataStore3 was parsed resulting
-				// in a notification to GC of reference to dataStore3 and this event would be logged in error.
-				mockLogger.assertMatchNone(
-					[
-						{
-							eventName:
-								"fluid:telemetry:ContainerRuntime:GarbageCollector:GC_Tombstone_DataStore_Revived",
-						},
-					],
-					"Revived event not as expected",
-				);
+				// There will be revived events because of auto-recovery. However, they should all have "autorecovery"
```

Comment:
I feel like this test doesn't test what it says anymore, it just tests autorecovery. Double-check that there's nothing novel here, but I think you can just delete these tests.  They were regression/bug tests before Op-Based-Reference work went out, I think?

Reply:
Yeah. It doesn't add much value. It only tests now that we don't see revived events because of notifications from DDS but it's kinda obvious.